spec/controllers/reservations_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+include AdminSpecHelper
+include LoginHelper
```

Comment:
Probably should do the reverse and just remove the login_as methods from admin helper